### ISSUE TITLE
Unbuildable against Linux 6.8+

### ISSUE THREAD

**Issue by nabijaczleweli** [t2mune/nield#3]
```
$ git llog -- ./include/uapi/linux/tc_act/tc_ipt.h
commit ba24ea129126362e7139fed4e13701ca5b71ac0b
Author: Jamal Hadi Salim <jhs@mojatatu.com>
Date:   Thu Dec 21 16:31:03 2023 -0500

    net/sched: Retire ipt action

    The tc ipt action was intended to run all netfilter/iptables target.
    Unfortunately it has not benefitted over the years from proper updates when
    netfilter changes, and for that reason it has remained rudimentary.
    Pinging a bunch of people that i was aware were using this indicates that
    removing it wont affect them.
    Retire it to reduce maintenance efforts. Buh-bye.
```
and patching the #include out yields
```
gcc -DHAVE_CONFIG_H -I. -I..   -Wdate-time -D_FORTIFY_SOURCE=2  -g -O2 -Werror=implicit-function-declaration -ffile-prefix-map=/srv/nield=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection -c -o tcmsg_qdisc_cbq.o tcmsg_qdisc_cbq.c
tcmsg_qdisc_cbq.c: In function 'parse_tca_options_cbq':
tcmsg_qdisc_cbq.c:29:24: error: '__TCA_CBQ_MAX' undeclared (first use in this function); did you mean '__TCA_CBS_MAX'?
   29 |     struct rtattr *cbq[__TCA_CBQ_MAX];
      |                        ^~~~~~~~~~~~~
      |                        __TCA_CBS_MAX
tcmsg_qdisc_cbq.c:29:24: note: each undeclared identifier is reported only once for each function it appears in
tcmsg_qdisc_cbq.c:36:30: error: 'TCA_CBQ_MAX' undeclared (first use in this function); did you mean 'TCA_CBS_MAX'?
   36 |     parse_nested_rtattr(cbq, TCA_CBQ_MAX, tca);
      |                              ^~~~~~~~~~~
      |                              TCA_CBS_MAX
tcmsg_qdisc_cbq.c:38:12: error: 'TCA_CBQ_LSSOPT' undeclared (first use in this function)
   38 |     if(cbq[TCA_CBQ_LSSOPT]) {
      |            ^~~~~~~~~~~~~~
```
and these macros are no longer in sid linux-libc-dev (6.10.11-1).

**Comment by t2mune** [comment]
I have fixed it, can you please try again with v0.6.2?

**Comment by nabijaczleweli** [comment]
Confirm fixed in v0.6.2.
